src/Microsoft.Extensions.Configuration.AzureAppConfiguration/IConfigurationClientProvider.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+
+using Azure.Data.AppConfiguration;
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration
+{
+    internal interface IConfigurationClientProvider
+    {
+        IEnumerator<ConfigurationClient> GetClientEnumerator();
```

Comment:
I think a more expected .NET pattern would be `IEnumerable<ConfigurationClient> GetClients()`

Reply:
Done!